packages/playground/src/components/user-management/users-list.tsx

```diff
@@ -0,0 +1,297 @@
+  const handleReset = () => {
+    dispatch({ type: 'START_RESETTING' })
+    setTimeout(() => {
+      dispatch({ type: 'RESET_PASSWORD_SUCCESS' })
+      setTimeout(() => {
+        closeDialog('resetPassword')
+        dispatch({ type: 'RESET_PASSWORD_RESET' })
+      }, 2000)
+    }, 2000)
+  }
+
+  const moreActionsTooltip = ({ user }: { user: UsersProps }) => {
+    return (
+      <DropdownMenu>
+        <DropdownMenuTrigger asChild>
+          <Button variant="ghost" size="xs">
+            <Icon name="vertical-ellipsis" size={14} className="text-tertiary-background" />
+          </Button>
+        </DropdownMenuTrigger>
+        <DropdownMenuContent className="shadow-sm py-2 bg-primary-background border border-gray-800 rounded-[10px] w-[180px]">
+          <DropdownMenuGroup>
+            <DropdownMenuItem
+              className="cursor-pointer"
+              onSelect={() => {
+                openDialog('removeAdmin', user)
+              }}>
+              <DropdownMenuShortcut className="ml-0">
+                <Icon name="trash" className="mr-2" />
+              </DropdownMenuShortcut>
+              Remove Admin
+            </DropdownMenuItem>
+            <DropdownMenuItem
+              className="cursor-pointer"
+              onSelect={() => {
+                openDialog('resetPassword', user)
+              }}>
+              <DropdownMenuShortcut className="ml-0">
+                <Icon name="cog-6" className="mr-2" />
+              </DropdownMenuShortcut>
+              Reset Password
+            </DropdownMenuItem>
+            <DropdownMenuItem
+              className="cursor-pointer"
+              onSelect={() => {
+                openDialog('edit', user)
+              }}>
+              <DropdownMenuShortcut className="ml-0">
+                <Icon name="edit-pen" className="mr-2" />
+              </DropdownMenuShortcut>
+              Edit User
+            </DropdownMenuItem>
+            <DropdownMenuSeparator />
+            <DropdownMenuItem
+              className="cursor-pointer text-red-400 hover:text-red-400 focus:text-red-400"
+              onSelect={() => {
+                openDialog('delete', user)
+              }}>
+              <DropdownMenuShortcut className="ml-0">
+                <Icon name="trash" className="mr-2 text-red-400" />
+              </DropdownMenuShortcut>
+              Delete User
+            </DropdownMenuItem>
+          </DropdownMenuGroup>
+        </DropdownMenuContent>
+      </DropdownMenu>
+    )
+  }
+
+  return (
+    <>
+      <Table variant="asStackedList">
+        <TableHeader>
+          <TableRow>
+            <TableHead className="text-primary">Name</TableHead>
+            <TableHead className="text-primary">Email</TableHead>
+            <TableHead className="text-primary">Display Name</TableHead>
+            {users[0]?.created && <TableHead className="text-right text-primary">Date added</TableHead>}
+            <TableHead>
+              <></>
```

Comment:
@cjlee01 Do you mean the `users[0]` or `<></>TableHead`? 
If you are asking about `<></>` `tablehead`, it's because the table head of more actions in table list is empty context so we left the `<></>`  for breaking the position of table header
If it's about the users[0].created, I will confirm from the api call so will revise it during of api integration in another PR.

Reply:
<></>